.github/ISSUE_TEMPLATE/community-request-template.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Community Request Template
+about: Suggest an idea for this project
+title: ''
+labels: feature-request
+assignees: ''
+
+---
+
+# Community Note
+* Please vote for this issue by adding a 👍 reaction to the issue. This will help the community and maintainers from the Amazon Chime SDK team to prioritize this request.
+* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request.
+* Please go through the existing [issues](https://github.com/aws/amazon-chime-sdk-js/issues) to see if your request has already been recommended.
+* Please do not use this template for bugs or support. Each individual repository has a PR template, documentation and resources for troubleshooting (https://github.com/aws/amazon-chime-sdk-js#troubleshooting-and-support).
```

Comment:
Should this be?
troubleshooting (https://github.com/aws/amazon-chime-sdk-js#troubleshooting-and-support) -> [troubleshooting](https://github.com/aws/amazon-chime-sdk-js#troubleshooting-and-support).

Basically markdown the troubleshooting as it is having an extra space just before the link.